vochain/scrutinizer/process.go

```diff
@@ -153,20 +152,24 @@ func (s *Scrutinizer) ProcessList(entityID []byte,
 	return procs, err
 }
 
-// ProcessCount return the number of processes indexed
-func (s *Scrutinizer) ProcessCount(entityID []byte) int64 {
+// ProcessCount returns the number of processes indexed
+func (s *Scrutinizer) ProcessCount(entityID []byte) uint64 {
 	startTime := time.Now()
 	defer func() { log.Debugf("ProcessCount took %s", time.Since(startTime)) }()
 	var c uint32
 	var err error
 	if len(entityID) == 0 {
-		// If no entity ID, return the cached count of all processes
-		return atomic.LoadInt64(&s.countTotalProcesses)
+		// If no entity ID, return the stored count of all processes
+		processCountStore := &indexertypes.CountStore{}
+		if err = s.db.Get(indexertypes.CountStoreProcesses, processCountStore); err != nil {
+			log.Warnf("could not get the process count: %v", err)
```

Comment:
That should be an `Error()` since its something that should never happen (actually it should be a Panic() but in general we are not making the components Panic)